setup.py

```diff
@@ -31,8 +31,8 @@ def run(self):
 # "READER_TO_INSTALL" lookup table from aicsimageio/formats.py.
 format_libs: Dict[str, List[str]] = {
     "base-imageio": [
-        "imageio[ffmpeg]>=2.11.0",
-        "Pillow>=9.3",
+        "imageio[ffmpeg]>=2.11.0,<2.28.0",
```

Comment:
Ah I see what you meant by Pillow plug-in your in Slack message now. Bummer that this is now also an issue :( seems like it can be handled outside of this PR though